### PR TITLE
Added excel_ui.add_beads_stats() function.

### PR DESCRIPTION
**Pull request by castillohair**

### CODE REVIEW

File: FlowCal/excel_ui.py [hunk 1]
```diff
@@ -562,6 +562,53 @@ def process_samples_table(samples_table,
 
     return samples
 
+def add_beads_stats(beads_table, beads_samples):
```
Comment (JS3xton) [inline FlowCal/excel_ui.py:569]: I like that this function name is verbose and consistent with other function names (e.g. `process_beads_table`). Might consider updating `add_stats` to `add_samples_stats`.

File: FlowCal/excel_ui.py [hunk 2]
```diff
@@ -808,9 +855,14 @@ def run(verbose=True, plot=True):
         plot=plot,
         plot_dir='plot_samples')
 
-    # Add stats to samples table
+    # Add stats to beads table
     if verbose:
         print ""
+        print("Calculating statistics for beads...")
+    add_beads_stats(beads_table, beads_samples)
```
Comment (JS3xton) [inline FlowCal/excel_ui.py:863]: i think old print statement above (`print ""`) is not compliant with Python 3.

File: FlowCal/excel_ui.py [hunk 3]
```diff
@@ -562,6 +562,53 @@ def process_samples_table(samples_table,
...
+    beads_table['Number of Events'] = [sample.shape[0]
+                                       for sample in beads_samples]
+    beads_table['Acquisition Time (s)'] = [sample.acquisition_time
+                                           for sample in beads_samples]
+
+    # List of channels that require stats columns
+    r = re.compile(r'^\s*(\S*)\s*MEF\s*Values\s*$')
+    headers = list(beads_table.columns)
+    stats_headers = [h for h in headers if r.match(h)]
+    stats_channels = [r.search(h).group(1) for h in stats_headers]
```
Comment (JS3xton) [inline FlowCal/excel_ui.py]: this obscure regex code has been invoked multiple times now (up to 5 by my count?). Might consider pulling it out as a module-level function? maybe hidden?

**Comment by JS3xton** [comment]
Didn't run this, just looked at the source. Minor comments, but otherwise looks good to me if someone else has checked that it produces the right output.

**Comment by JS3xton** [comment]
related to #175

File: FlowCal/excel_ui.py (hunk 2 above)
Comment (castillohair) [inline FlowCal/excel_ui.py:863]: Oops. Fixed in a5cd6b6282b4d65007095501ad5a78f6c159477b.

File: FlowCal/excel_ui.py (hunk 1 above)
Comment (castillohair) [inline FlowCal/excel_ui.py:569]: Done in e383690f453d761903ab43f55417c12f7b844f18.

File: FlowCal/excel_ui.py (hunk 3 above)
Comment (castillohair) [inline FlowCal/excel_ui.py]: They're two different regexes, actually. Moved them as module-level variables in bd122e8d09088028a2f78f715b7c098d9242454f.

**Pull request merged**